deploy.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
 
-if [ -z ${1+x} ]; then SOURCE_BRANCH='master'; else SOURCE_BRANCH=$1; fi
+if [ -z ${1+x} ]; then DEPLOY_TYPE='patch'; else DEPLOY_TYPE=$1; fi
+# Do we need the ability to push other branches to production? Seems _really_ dangerous to me
```

Comment:
I think how dangerous it is depends on the branch. But yea, I have found it useful in the past to be able to do that.